profiles/api_test.py

```diff
@@ -0,0 +1,73 @@
+"""
+Tests for profile functions
+"""
+
+from mock import Mock
+
+from django.db.models.signals import post_save
+from django.test import TestCase
```

Comment:
FYI you will see `from unittest import TestCase` in certain tests. Django's `TestCase` adds on some useful things like database setup and teardown and a test `Client` to do fake requests.